src/plugins/kibana_react/public/app_links/redirect_app_link.test.tsx

```diff
@@ -0,0 +1,243 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { MouseEvent } from 'react';
+import { mount } from 'enzyme';
+import { applicationServiceMock } from '../../../../core/public/mocks';
+import { RedirectAppLinks } from './redirect_app_link';
+import { BehaviorSubject } from 'rxjs';
+
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+describe('RedirectAppLinks', () => {
+  let application: ReturnType<typeof applicationServiceMock.createStartContract>;
+
+  beforeEach(() => {
+    application = applicationServiceMock.createStartContract();
+    application.parseAppUrl.mockReturnValue({ app: 'targetApp' });
```

Comment:
It doesn't appear this is used anymore? Can we remove this API now?

Reply:
It isn't. I feel like I'll have to reimplement it soon enough by doing so, but you're probably right, I should remove it from this PR.